package.json

```diff
@@ -55,17 +55,17 @@
     "@synthetixio/deps": "workspace:*",
     "@types/mocha": "^10.0.3",
     "@types/node": "^18.18.8",
-    "@typescript-eslint/eslint-plugin": "^6.9.1",
-    "@typescript-eslint/parser": "^6.9.1",
-    "@usecannon/cli": "^2.9.3",
-    "eslint": "^8.52.0",
+    "@typescript-eslint/eslint-plugin": "^6.10.0",
+    "@typescript-eslint/parser": "^6.10.0",
+    "@usecannon/cli": "^2.9.7",
+    "eslint": "^8.53.0",
     "eslint-plugin-no-only-tests": "^3.1.0",
     "lint-staged": "^15.0.2",
     "mocha": "^10.2.0",
     "mocha-junit-reporter": "^2.2.1",
     "pre-commit": "^1.2.2",
     "prettier": "^3.0.3",
-    "prettier-plugin-solidity": "^1.1.3",
+    "prettier-plugin-solidity": "^1.1.4-dev",
```

Comment:
This one causing ci to fail maybe?

Reply:
fixed. yep. they shouldn't have published it under `latest` tag 🤦